it-Italian/Localizable.strings

```diff
@@ -0,0 +1,298 @@
+"what_you_can_do_with_aliases_c_title" = "C. Gestisci il tuo business con nuovi alias";
+"what_you_can_do_with_aliases_c_description" = "Utilizza alias come business email\n\nRisparmia fino a €5 al mese per ogni alias creato tramite SimpleLogin.";
+"our_team" = "Il nostro team";
+"contact_us" = "Contattaci";
+"pricing" = "Prezzi";
+"blog" = "Blog";
+"help" = "Aiuto";
+"roadmap" = "Roadmap";
+"terms_and_conditions" = "Termini e Condizioni";
+"privacy_policy" = "Regole sulla privacy";
+// Log in
+"connecting_to_server" = "Connessione al server...";
+"email_address" = "Indirizzo email";
+"invalid_email_address" = "Indirizzo email non valido";
+"password" = "Password";
+"sign_in" = "Sign in";
+"forgot_password" = "Password dimenticata";
+"forgot_password_explication" = "Per ragioni di sicurezza tutte le richieste di \"Password dimenticata\" saranno validate internamete e l'utente non sarà informato se la richiesta è andata a buon fine o meno.\n\nControlla che hai inserito l'indirizzo email giusto prima di procedere.";
+"reset_password" = "Cambia password";
+"reset_password_success" = "Abbiamo inviato una email per cambiare la tua password a \"%@\"";
+"sign_in_with_api_key" = "Sign in con chiave API";
+"enter_api_key" = "Inserisci chiave API";
+"enter_api_key_explication" = "Per gestire la tua chiave API devi fare login in SimpleLogin tramite il browser e poi cliccare sul tuo nome ";
+"set_api_key" = "Configura chiave API";
+"change_api_url" = "Cambia API URL";
+"change_api_url_warning" = "NON CAMBIARE la API URL a meno che non stai facendo self-hosting di SimpleLogin sul tuo server personale. Nella maggior parte dei casi questa configurazione deve essere lasciata a default.";
+"change_api_url_default_value" = "L'impostazione di default è https://app.simplelogin.io";
+"current_api_url" = "API URL corrente";
+"empty_api_url" = "API URL è vuota";
+"apply_change" = "Conferma modifica";
+"reset_to_default_value" = "Reimposta con valore di default";
+"changed_api_url_to" = "API URL cambiata con %@";
+"reset_api_url_to" = "Reimposta API URL con https://app.simplelogin.io";
+ // Sign up
+"sign_up" = "Registrati";
+"sign_up_for_simplelogin" = "Registrati con SimpleLogin";
+"sign_up_success" = "Controlla la tua email per il codice di verifica";
+"minimum_8_characters" = "Minimo 8 caratteri";
+// Verification
+"enter_otp" = "Inserisci OTP";
+"enter_activation_code" = "Inserisci codice di verifica";
+"paste_and_verify" = "Incolla & verifica";
+"code_found_from_the_clipboard" = "\"%@\" trovato nella clipboard";
+"wrong_code_too_many_times" = "Codice errato inserito troppe volte";
+"send_new_activation_code" = "Ti invieremo un nuovo codice di verifica a \"%@\"";
+"check_new_verification_code" = "Controlla la tua email per il nuovo codice di verifica";
+// Time
+<strind name="created_on" = "Creato";
```

Comment:
Syntax error here.

Reply:
thanks, that's fixed now. The same issue was on the original EN files, so I've amended there too.

Reply:
thank you!

Reply:
That's because I started with the .xml file and do some find/replace to make the .strings file 😅